ci-operator/jobs/redhat-openshift-ecosystem/provider-certification-tool/redhat-openshift-ecosystem-provider-certification-tool-main-periodics.yaml

```diff
@@ -80,6 +80,81 @@ periodics:
     - name: result-aggregator
       secret:
         secretName: result-aggregator
+- agent: kubernetes
+  cluster: vsphere
+  cron: 0 0 * * 2
+  decorate: true
+  decoration_config:
+    skip_cloning: true
+  extra_refs:
+  - base_ref: main
+    org: redhat-openshift-ecosystem
+    repo: provider-certification-tool
+  labels:
+    ci-operator.openshift.io/cloud: vsphere
+    ci-operator.openshift.io/cloud-cluster-profile: vsphere
+    ci-operator.openshift.io/variant: "4.13"
+    ci.openshift.io/generator: prowgen
+    job-release: "4.13"
+    pj-rehearse.openshift.io/can-be-rehearsed: "true"
+  name: periodic-ci-redhat-openshift-ecosystem-provider-certification-tool-main-4.13-provider-certification-tool-vsphere-upi-platform-external
```

Comment:
Can we enable the Slack notifications as we used to do in the other [two jobs](https://github.com/openshift/release/blob/021658c71bb2eb8783c78f259ac0a13f4e79f7e1/ci-operator/jobs/redhat-openshift-ecosystem/provider-certification-tool/redhat-openshift-ecosystem-provider-certification-tool-main-periodics.yaml#L174-L180)?

Reply:
Done. I also altered report templates for other jobs to contain links to a specific run.

PTAL https://github.com/openshift/release/pull/36572/commits/bfeb4b050f9ba4fd00a5b4945f2f8c1635886264